main.py

```diff
@@ -0,0 +1,106 @@
+from fastapi import FastAPI, Depends, HTTPException
+from fastapi_pagination import add_pagination, Page, paginate
+from sqlalchemy.orm import Session
+from http import HTTPStatus
+
+import crud
+import schemas
```

Comment:
do you really like to write those `schema` prefixes before every Pydantic model?